apps/docs/docgen-fix-plugin.js

```diff
@@ -0,0 +1,73 @@
+const path = require('path');
+const { glob } = require('glob');
+const { withCustomConfig } = require('react-docgen-typescript');
+
+/**
+ * Custom docgen plugin that parses component files individually to prevent
+ * generic type resolution conflicts that occur when parsing barrel exports.
+ *
+ * This plugin replaces docusaurus-plugin-react-docgen-typescript with a version
+ * that correctly handles generic types like ValueBase<T> by parsing each component
+ * file separately instead of all at once.
+ */
+module.exports = function customDocgenPlugin(context, options) {
+  const { src, parserOptions: userParserOptions } = options;
+
+  return {
+    name: 'docusaurus-plugin-react-docgen-typescript',
+    async loadContent() {
+      const tsconfigPath = path.join(__dirname, '../../apps/storybook/tsconfig.storybook.json');
+
+      const parserOptions = {
+        propFilter: prop => {
+          if (prop.parent) {
+            return !prop.parent.fileName.includes('@types/react');
```

Comment:
```suggestion
            return !(
             prop.parent.fileName.includes('@types/react') ||
             prop.parent.fileName.includes('dom.d.ts')
           )
```
Vi kanske ska fippla tillbaka den där

Reply:
Ja sant vi kanske ska fippla över den på storybook också då?

Reply:
Yes!